lib/preneeds/logged_configuration.rb

```diff
@@ -0,0 +1,27 @@
+# frozen_string_literal: true
+
+require 'common/client/middleware/logging'
+
+module Preneeds
+  class LoggedConfiguration < Configuration
+    def connection
+      path = Preneeds::Configuration.url
+      @faraday ||= Faraday.new(
+        path, headers: base_request_headers, request: request_options, ssl: { verify: false }
+      ) do |conn|
+        conn.options.timeout = TIMEOUT
+
+        conn.request :soap_headers
+
+        conn.response :preneeds_parser
+        conn.response :soap_parser
+        conn.response :eoas_xml_errors
+        conn.response :clean_response
+
+        conn.use :breakers
```

Comment:
Just a heads up here, that breakers is going to use a short code for determining what the service is, so this connection might get lumped in with another connection. That may have unintended consequences in the event that one connection is different than another, but I guess since the path is the same it is probably not a problem.